doc/administrator/editing.rst

```diff
@@ -3,12 +3,10 @@
 Editing
 =======
 
-This section describes how to set up feature editing in c2cgeoportal
-applications.
+This section explains how to configure the edit function in c2cgeoportal.
 
-Just like most administrative tasks setting up editing in a c2cgeoportal
-application involves intervening in the database, through the c2cgeoportal
-administration interface.
+Just like most administrative tasks the configuration of the edition in the c2cgeoportal application
```

Comment:
Suggestion: "Just like most admininistrative tasks, the configuration of the editing functionality involves intervening in the database via the c2cgeoportal administration interface."